src/interf.jl

```diff
@@ -35,6 +35,8 @@ function nnmf(X::AbstractMatrix{T}, k::Integer;
         alginst = MultUpdate{T}(obj=:mse, maxiter=maxiter, tol=tol, verbose=verbose)
     elseif alg == :multdiv
         alginst = MultUpdate{T}(obj=:div, maxiter=maxiter, tol=tol, verbose=verbose)
+    elseif alg == :cd
```

Comment:
Can you add a test for calling `nnmf` with `alg=:cd`?

Reply:
I added a test (revealing a bug, thanks!), also added interf.jl test to the list of tests performed. This in turn revealed a deprecation in projals.jl (vecnorm to norm) which I also had to fix in order for the tests to pass.